src/map/lua/lua_baseentity.cpp

```diff
@@ -1517,6 +1517,36 @@ uint8 CLuaBaseEntity::getCurrentAction()
     return action;
 }
 
+/************************************************************************
+ *  Function: canUseAbilities()
+ *  Purpose : Sees if a mob has perticular effects that stop it from doing thing
+ *  Example : mob:canUseAbilities()
+ *  Notes   : Can't use on NPCs
+ ************************************************************************/
+
+bool CLuaBaseEntity::canUseAbilities()
+{
+    XI_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
+
+    if (m_PBaseEntity == nullptr)
```

Comment:
m_PBaseEntity should never be null in these functions.  @zach2good correct me if I'm wrong, but I believe there is safety in the CLuaBaseEntity class itself.

Reply:
Awesome i'll adjust on confirm, I just didn't want to introduce null chances.

Reply:
Winter confirmed what you said @claywar - null check removed.